packages/eslint-plugin-svelte/src/types.ts

```diff
@@ -108,6 +109,18 @@ export interface PartialRuleMetaData {
 	deprecated?: boolean;
 	replacedBy?: string[] | { note: string };
 	type: 'problem' | 'suggestion' | 'layout';
+	/**
+	 * Conditions to determine whether this rule should be applied.
+	 * Multiple conditions can be specified as array, and the rule will be applied if any one of them matches (logical OR).
+	 * If not specified, the rule will be applied to all files.
+	 */
+	conditions?: {
+		svelteVersion?: string; // e.g.: '>= 5.0.0'
```

Comment:
I'm a little bit torn about these version constraints - aren't we trying to be too clever here? Maybe this encourages rules that rely on specific things being in specific versions, which IMHO is not a good design... I think I would prefer to have a more limited way of conditioning rules based on Svelte versions - probably just based on major versions, or maybe even version 5 vs. versions 3/4 (just 2 options)...

Reply:
Agreed and updated: https://github.com/sveltejs/eslint-plugin-svelte/pull/980/commits/910effd2057a67921a6ed0b01be5de31aae52d4a